File: PlayerBuild/PassiveSkills.cs
using Newtonsoft.Json.Linq;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using TT2Advisor.Import;

namespace TT2Advisor.PlayerBuild
{
    public class PassiveSkills
    {
        //public PassiveSkills(JToken passiveSkillsToken)
        //{
        //    IntimidatingPresence = (int)passiveSkillsToken.SelectToken("['Intimidating Presence']");
        //    PowerSurge = (int)passiveSkillsToken.SelectToken("['Power Surge']");
        //    AntiTitanCannon = (int)passiveSkillsToken.SelectToken("['Anti-Titan Cannon']");
        //    MysticalImpact = (int)passiveSkillsToken.SelectToken("['Mystical Impact']");
        //    ArcaneBargain = (int)passiveSkillsToken.SelectToken("['Arcane Bargain']");
        //    SilentMarch = (int)passiveSkillsToken.SelectToken("['Silent Marth']");
        //    CloakAndDagger = (int)passiveSkillsToken.SelectToken("['Cloak And Dagger']");
        //}

        public PassiveSkills(Import.PassiveSkills passiveSkills)
        {
            IntimidatingPresence = passiveSkills.IntimidatingPresence;
            PowerSurge = passiveSkills.PowerSurge;
            AntiTitanCannon = passiveSkills.AntiTitanCannon;
            MysticalImpact = passiveSkills.MysticalImpact;
            ArcaneBargain = passiveSkills.ArcaneBargain;
            SilentMarch = passiveSkills.SilentMarch;
            CloakAndDagger = passiveSkills.CloakAndDagger;
        }

        public int IntimidatingPresence { get; set; }
        public int PowerSurge { get; set; }
        public int AntiTitanCannon { get; set; }
        public int MysticalImpact { get; set; }
        public int ArcaneBargain { get; set; }
        public int SilentMarch { get; set; }
        public int CloakAndDagger { get; set; }
    }
}

File: TT2Advisor.UI/ClientApp/src/components/myBuild/passiveSkills.js
import React, { Component } from 'react';

export class PassiveSkills extends Component {

    passiveSkills: any;

    constructor(props) {
        super(props);
        this.state = { passiveSkills: this.props.passiveSkills, loading: true };
    }

    componentDidMount() {
        
    }

    render() {
        let contents = this.state.loading
            ? <p><em>Loading...</em></p>
            : PassiveSkills.renderPassiveSkills(this.state.passiveSkills);

        return (
            <div>
                <h2>Passive Skills</h2>
                <table className='table table-striped' aria-labelledby="tabelLabel">
                    <thead>
                        <tr>
                            <th>Stat</th>
                            <th>Value</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td>Intimidating Presence</td>
                            <td>{this.state.passiveSkills.intimidatingPresence}</td>
                        </tr>
                        <tr>
                            <td>Power Surge</td>
                            <td>{this.state.passiveSkills.powerSurge}</td>
                        </tr>
                        <tr>
                            <td>Anti-Titan Cannon</td>
                            <td>{this.state.passiveSkills.antiTitanCannon}</td>
                        </tr>
                        <tr>
                            <td>Mystical Impact</td>
                            <td>{this.state.passiveSkills.mysticalImpact}</td>
                        </tr>
                        <tr>
                            <td>Arcane Bargain</td>
                            <td>{this.state.passiveSkills.arcaneBargain}</td>
                        </tr>
                        <tr>
                            <td>Silent March</td>
                            <td>{this.state.passiveSkills.silentMarch}</td>
                        </tr>
                        <tr>
                            <td>Cloak And Dagger</td>
                            <td>{this.state.passiveSkills.cloakAndDagger}</td>
                        </tr>
                    </tbody>
                </table>
            </div>
        );
    }
}
